nixos/modules/services/networking/flannel.nix

```diff
@@ -144,6 +144,7 @@ in {
       description = "Flannel Service";
       wantedBy = [ "multi-user.target" ];
       after = [ "network.target" ];
+      requires = [ "network.target" ];
```

Comment:
`requires=network.target` is almost never what you want.  systemd even says you should avoid it

See the classic https://www.freedesktop.org/wiki/Software/systemd/NetworkTarget/ page.

Reply:
Okidoki, will remove!